js/src/views/Account/account.js

```diff
@@ -50,15 +51,20 @@ class Account extends Component {
     balances: PropTypes.object,
     certifications: PropTypes.object,
     netVersion: PropTypes.string.isRequired,
+    newError: PropTypes.func,
     params: PropTypes.object
   }
 
   store = new Store();
   hwstore = HardwareStore.get(this.context.api);
 
   componentDidMount () {
+    const { accounts, newError, params } = this.props;
+    const { address } = params;
+
     this.props.fetchCertifiers();
     this.setVisibleAccounts();
+    this.store.insertProps(this.context.api, accounts, address, newError);
```

Comment:
As per above comment and the use of the stores. In this case, I would not change the store, rather use the already created ExportStore to export the individual accounts.